### PR TITLE
TelegramClientUpdates public field to Property.

### PR DESCRIPTION
**Pull request by GihanSoft**
`TelegramClientUpdates` public field to Property.

### CODE REVIEW

**Comment by GihanSoft** [comment]
why this have previous commits? @ilyalatt

**Comment by ilyalatt** [comment]
you need to pull master and merge master into this branch

File: Telega/TelegramClient.cs [hunk 1]
```diff
@@ -42,8 +42,7 @@ ISessionStore sessionStore
             Updates = new TelegramClientUpdates(_bellhop);
         }
 
-        public void Dispose()
-        {
+        public void Dispose() {
```
Comment (GihanSoft) [inline Telega/TelegramClient.cs:45]: to be same as others

**Comment by GihanSoft** [comment]
@ilyalatt tanks.

File: Telega/TelegramClient.cs (hunk 1 above)
Comment (ilyalatt) [reply Telega/TelegramClient.cs:45]: if so we'll need to use this style for all methods
i use this style only if i have a method formatted like

```
void F(
    int a,
    int b
) {
    ...
}
```

can it be configured in .editorconfig?

**Comment by ilyalatt** [comment]
Can you make this change to all public readonly fields to reduce number of PR's?

**Comment by GihanSoft** [comment]
@ilyalatt hmm. I'll try

**Comment by GihanSoft** [comment]
** Code gen need fix? **
I think it's fixed but check it. I don't know much about code gen.
need extra fix for in line initializer like this
```csharp
public CustomObservable<UpdatesType> Updates { get; } = new CustomObservable<UpdatesType>();
```

you can apply it with this regex replace:
search pattern:
```regex
public readonly (.+?);
```

replace pattern:
```regex
public $1 { get; }
```
2536 replace
First time my CPU worked full power more than 1m in programming ant it wasn't a crash.

**Comment by ilyalatt** [comment]
Looks good. Fix `Dispose` brace and I'll merge it.

**Comment by GihanSoft** [comment]
where?

**Comment by ilyalatt** [comment]
i'll fix it myself

**Pull request merged**